### DISCUSSION TITLE
Add Oxygen Gear for Pets

### DISCUSSION THREAD

**Discussion by vos6434** [TeamGalacticraft/Galacticraft#171] [Feature Requests]
shift right-click to equip oxygen gear for a pet.
shift right-click on the pet again to see the remaining oxygen.

- [x] Parrot
- [x] Wolf
- [x] Cat

**Comment by Atakku** [comment]
- [ ] Horses
- [ ] Donkeys
- [ ] Mules
- [ ] Llamas
- [ ] Foxes

**Comment by EzerArch** [comment]
Ok, but do we have rockets that transport animals?

(assuming the player is not playing with other mods that can capture animals)

**Comment by Atakku** [comment]
~~Just yank the pet to the rocket using a leash and call it a day~~

**Comment by CheetahCyberBoi** [comment]
Maybe only allow light O2 tanks for it because bigger ones would just be cruel.

**Comment by ScoutTheSaneless** [comment]
@Atakku on ur pet list additions, FOXES CANNOT BE TAMED! The fox is unable to be tamed, only gain its trust which the game treats as separate instances. You cannot make a fox sit, nor can you use it for transport, ergo it is not tamable. The breeding method only results in the offspring fully trusting the player who bred the parents, which as I'll say again, the game treats differently in comparison to a tamed mob.

**Comment by ScoutTheSaneless** [comment]
Ik this sounds harsh, but ive seen SO many ppl get mixed up in thinking foxes r tamable when they're not...

**Comment by Atakku** [comment]
> Ik this sounds harsh, but ive seen SO many ppl get mixed up in thinking foxes r tamable when they're not...

Yeah I didn't know foxes weren't tamable untill a couple weeks ago

**Comment by Roelymole** [comment]
Merged in #468

<img width="1920" height="1080" alt="2025-07-11_00 44 18" src="https://github.com/user-attachments/assets/38247621-5ef9-45ec-b1d7-69d6f084b1de" />
<img width="1920" height="1080" alt="2025-07-11_00 46 41" src="https://github.com/user-attachments/assets/cbf94edf-bab3-4a5a-8090-88412de3d45e" />

**Comment by vos6434** [comment]
I think this discussion is good to close now.
